include/xrpl/protocol/Fees.h

```diff
@@ -37,6 +37,7 @@ struct Fees
     std::uint32_t extensionComputeLimit{
         0};  // Extension compute limit (instructions)
     std::uint32_t extensionSizeLimit{0};  // Extension size limit (bytes)
+    std::uint32_t gasPrice{0};            // price of WASM gas (micro-drops)
```

Comment:
All these names doesn't show any relations among them.  Consider to give them some common root / field of usage, for example 
```
extensionSizeLimit-> contractSizeLimit
extensionComputeLimit -> contractGasLimit
gasPrice -> contractGasPrice
ComputationAllowance -> contractGasPayment
sfFinishFunction -> sfContract
tefNO_WASM -> tefNO_CONTRACT
```

Reply:
Extensions != contracts, so that's an incorrect statement. More specifically, we may want different values for extensions vs contracts further down the line.

Reply:
What "extension" is refer to?

Reply:
https://dev.to/ripplexdev/a-proposed-vision-for-xrp-ledger-programmability-1gdj